### PR TITLE
Rename Indent Left/Right options in the Script Editor to Indent/Unindent

### PR DESCRIPTION
**Pull request by MewPurPur**
Seems just better as it doesn't make you think about directions, also doesn't assume LTR.

### CODE REVIEW

File: editor/plugins/script_text_editor.cpp [hunk 1]
```diff
@@ -2158,8 +2158,8 @@ void ScriptTextEditor::register_editor() {
 	// Leave these at zero, same can be accomplished with tab/shift-tab, including selection.
 	// The next/previous in history shortcut in this case makes a lot more sense.
 
-	ED_SHORTCUT("script_text_editor/indent_left", TTR("Indent Left"), Key::NONE);
-	ED_SHORTCUT("script_text_editor/indent_right", TTR("Indent Right"), Key::NONE);
+	ED_SHORTCUT("script_text_editor/dedent", TTR("Dedent"), KeyModifierMask::SHIFT | Key::TAB);
+	ED_SHORTCUT("script_text_editor/indent", TTR("Indent"), Key::NONE);
```
Comment (akien-mga) [inline editor/plugins/script_text_editor.cpp:2162]: I think "indent" should come first (here and in the menus).
It made sense to have left before right (at least for LTR centric users), but not to have dedent before indent.

**Comment by akien-mga** [comment]
"dedent" or "unindent"?

Python seems to use the former but many sources use the latter.

Google gives 173k hits for "dedent" (but many of those are hits to tooth removal services...) and 85k hits for "unindent".

Wiktionary lists ["unindent"](https://en.wiktionary.org/wiki/unindent) and ["outdent"](https://en.wiktionary.org/wiki/outdent) as antonyms for indent: https://en.wiktionary.org/wiki/indent
("outdent" is typographical term which doesn't match the programming meaning)
It also has an entry for ["dedent"](https://en.wiktionary.org/wiki/dedent) which it qualifies as "primiarly Python".

**Comment by MewPurPur** [comment]
Looks like Unindent is used in CodeEdit, dedent in String... I don't think consistency is that important here, at least enough to do things in beta.

Given that CodeEdit uses unindent, whaddya think of just using that?

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**